heliopy/data/artemis.py

```diff
@@ -10,9 +10,10 @@
 
 from heliopy.data import util
 from heliopy import config
+import pathlib
 
 data_dir = config['download_dir']
-themis_dir = data_dir + '/themis'
+themis_dir = data_dir / 'themis'
```

Comment:
😮 the only reason that this was broke without your change is because I didn't use `os.path.join` anyway here...

Reply:
Yep! It was just a small edit, but it seems Travis is also failing in some other module as well, which AppVeyeor isn't finding.

Reply:
https://travis-ci.org/heliopython/heliopy/jobs/347014411

Reply:
Appveyor doesn't run data tests that need a CDF library installed (because I haven't found a way of installing a CDF library on windows (which is why HelioPy should really transition to using a pure python CDF reader, but that's for another day))

Reply:
By the way, should 'Sunspot' be added into the changelog too?

Reply:
Yeah, that's on my todo list, but if you want to open a PR for that it would be welcome!